src/rules.js

```diff
@@ -1903,6 +1903,80 @@ module.exports = [
     },
     /**
      ==================================================================
+     TRANSITION
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        id: 'transition',
+        name: 'Transition',
+        prefix: '.Trs-',
+        properties: ['transition'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true
```

Comment:
I think people will use custom suffixes for complex values (i.e. multiple values separated by commas) but for the most common used case like the example I give above, people will want to simply pass a property and duration. Why forcing them to use that very same string as a custom suffix when Atomizer could do that for them *on the fly*?